curvefs/src/client/inode_wrapper.cpp

```diff
@@ -302,7 +312,13 @@ CURVEFS_ERROR InodeWrapper::UnLinkLocked(uint64_t parent) {
         //                      2. write "hello, world"
         //                      3. unlink this file
         //                      4. pread from 0 to 13, expected "hello, world"
-        AddVolumeExtentMapToInode();
+        auto err = FlushVolumeExtent();
```

Comment:
for correctness, if I issue an asynchronous update extent here, step 4 may still read nothing.

because unlink will remove current inode from the cache, so step 4 will fetch inode and extents from metaserver again.

so, if metaserver failover in this progress, current retry policy can't guarantee the update requests always succeeded before the fetch reqeusts.

and I think this problem also exists in other logic.